pages/common/rails.md

```diff
@@ -0,0 +1,23 @@
+# rails
+
+> Ruby on Rails, or Rails, is a server-side MVC framework written in Ruby under the MIT License.
+
+- Check current version of rails:
+
+`rails --version`
+
+- Create a new rails project:
+
+`rails new "{{name_of_project}}"`
+
+- Start local server for current project on port 3000 (while in projects root directory):
```

Comment:
It is well understood that you have to be in project's root directory to execute these commands. We have lot of pages like this and it's not explicitly mentioned anywhere. Unless a tool uses some other convention, we omit it for brevity. 

Let's remove it from all the descriptions.